analysis/import/SVNLogParser/src/test/kotlin/de/maibornwolff/codecharta/importer/svnlogparser/input/VersionControlledFileTest.kt

```diff
@@ -160,12 +159,12 @@ class VersionControlledFileTest {
         // then
         assertThat(versionControlledFile.filename).isEqualTo(oldFilename)
         assertThat(versionControlledFile.actualFilename).isEqualTo(filename)
-        assertThat(versionControlledFile.markedDeleted()).isFalse()
+        assertThat(versionControlledFile.markedDeleted()).isFalse
```

Comment:
Is this not a function call anymore?

Reply:
It is an attribute provided by the assertion, afaik